diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -178,6 +178,24 @@ def custom_error(self, msg):
         dest="plot",
         help="Do not show the plot.",
     )
+    group.add_option(
+        '--usefilenames',
+        action="store_true",
+        dest="usefilenames",
+        help="Use the file names as labels on plot."
+    )
+    group.add_option(
+        '--mlabel',
+        metavar="MLABEL",
+        dest="mlabel",
+        help="Set label for morphed data to MLABEL on plot. Ignored if using file names as labels.",
+    )
+    group.add_option(
+        '--tlabel',
+        metavar="TLABEL",
+        dest="tlabel",
+        help="Set label for target data to TLABEL on plot. Ignored if using file names as labels.",
+    )
```

Comment:
New option to set label names on plot.

Reply:
looks good, see comment.   Please put different issues on different PRs